### PR TITLE
OCPQE-14558: replace cert-manage with service mesh operator

### PR DESCRIPTION
**Pull request by jianzhangbjz**
The `cert-manager-operator` doesn't support the `AllNamespace` install mode anymore: https://github.com/openshift/cert-manager-operator/pull/83, so it will failed at 
```yaml
status:
  activeBundleDeployment:
    name: ""
  conditions:
  - lastTransitionTime: "2023-03-26T14:27:30Z"
    message: 'Failed to unpack the cert-manager-po-8v44b5 Bundle: convert registry+v1
      bundle to plain+v0 bundle: AllNamespace install mode must be enabled'
    reason: UnpackFailed
    status: "False"
    type: Installed
```
So, replace it with `amq-streams` operator.
```yaml
MacBook-Pro:~ jianzhang$ oc get platformoperator amq-streams-po  -o yaml
apiVersion: platform.openshift.io/v1alpha1
kind: PlatformOperator
metadata:
  creationTimestamp: "2023-03-29T07:05:33Z"
  generation: 1
  name: amq-streams-po
  resourceVersion: "211920"
  uid: b7c419f1-7ab7-42a2-89f4-1d118e6ba8fa
spec:
  package:
    name: amq-streams
status:
  activeBundleDeployment:
    name: amq-streams-po
  conditions:
  - lastTransitionTime: "2023-03-29T07:06:05Z"
    message: Successfully applied the amq-streams-po BundleDeployment resource
    reason: InstallSuccessful
    status: "True"
    type: Installed
```

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-aws-ipi-proxy-techpreview-po-p3-f28

File: ci-operator/step-registry/platformoperator/valid/amq-streams/platformoperator-valid-amq-streams-commands.sh [hunk 1]
```diff
@@ -6,11 +6,11 @@ cat <<EOF > ${SHARED_DIR}/manifest_cert_manager.yaml
 apiVersion: platform.openshift.io/v1alpha1
 kind: PlatformOperator
 metadata:
-  name: cert-manager-po
+  name: amq-streams-po
 spec:
   package:
-    name: openshift-cert-manager-operator
+    name: amq-streams
```
Comment (kuiwang02) [inline ci-operator/step-registry/platformoperator/valid/amq-streams/platformoperator-valid-amq-streams-commands.sh:12]: @jianzhangbjz `amq-streams` package is used by the case of logging team and netobserve team.
it possible cause conflicts when running case.

suggest to change to other operators.

**Comment by jianzhangbjz** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-proxy-techpreview-po-p3-f28

**Comment by kuiwang02** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-proxy-techpreview-p3-f28

**Comment by jianzhangbjz** [comment]
Installation pass. https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/37789/rehearse-37789-periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-aws-ipi-proxy-techpreview-p3-f28/1641017136090451968/artifacts/aws-ipi-proxy-techpreview-p3-f28/ipi-install-install/build-log.txt 
```yaml
level=info msg=Install complete!
level=info msg=To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=/tmp/installer/auth/kubeconfig'
level=info msg=Access the OpenShift web-console here: https://console-openshift-console.apps.ci-op-1j0z8y07-199ed.qe.devcluster.openshift.com
level=info msg=Time elapsed: 32m55s
Installer exit with code 0
```
Hi @kuiwang02 could you help approve it? Thanks!
/pj-rehearse ack

**Pull request merged**